README_WINDOWS.md

```diff
@@ -482,145 +348,83 @@ In order to get support for ASIO drivers, follow this directory structure:
-configuration files are located.
+Configuration and Building
+--------------------------
 
-Build SuperCollider.
+*Note:* This section assumes familiarity with the command line. The section below titled
+"Avoiding the command line" may be helpful if that doesn't describe you.
 
-#### Visual Studio 2013
+*Note:* You should always use forward slashes for paths passed to CMake.
 
-You may want to check the path where you installed Qt—if it's not `C:\Qt\`, then
-change these commands accordingly. You should `cd` into the SuperCollider source
-folder to begin.
+**Confirm the location of your Qt install before executing these commands.** You
+may need to modify them if you installed Qt somewhere else. The following
+commands should be executed starting from the root directory of the
+SuperCollider repository.
 
-    SET PATH=C:\Qt\5.5\msvc2013_64\bin;C:\Qt\Tools\msvc2013_64\bin;%PATH%
+    SET PATH=C:\Qt\5.5\msvc2013_64\bin;%PATH%
     SET CMAKE_PREFIX_PATH=C:\Qt\5.5\msvc2013_64
     mkdir build
     cd build
     cmake -G "Visual Studio 12 2013 Win64" ..
+    cmake --build . --config Release
 
-For the final step, you can either build within the Visual Studio IDE:
+For the final step, you can also build from within Visual Studio:
 
     start SuperCollider.sln
 
-Or build using `cmake` as usual:
-
-    cmake --build .
-
-The preceding MinGW section contains an explanation of this command sequence.
-
-#### MSYS2
-
-    mkdir build
-    cd build
-    cmake -G "MSYS Makefiles" ..
-    cmake --build . -- -j4
-
-The preceding MinGW section contains an explanation of this command sequence.
-`-- -j4` passes the argument -j4 (use 4 processor threads) to the makefile.
-Adjust the number to your processor ;)
-
-### Additional build settings
+Additional build settings
+-------------------------
 
 Listed here are common additional configurations you may want to use. For a
 more complete list, use `cmake --help` and/or see the section "More `cmake`:
 building supernova, qt-less, verbosity and more" below.
 
-#### `cmake` configuration options
+### Build type
 
-There are a number of additional options you can use in the penultimate build
-step:
+The `CMAKE_BUILD_TYPE` controls optimization level and whether or not debugging
+code (asserts, info messages, and other similar code) is included in the
+executable. The configuration is specified either in Visual Studio before
+building or in the build command given to CMake:
 
-    cmake -G "Toolchain Name" [-D OPTION_NAME] ..
+    cmake --build . --config <config>
 
-When using native MinGW or MSYS2 you can specify the build configuration with
-`CMAKE_BUILD_TYPE`. The default is `RelWithDebInfo` ("Release build, with debug
-info"), but there are other possibilities:
+The four options are:
 
-    cmake -D CMAKE_BUILD_TYPE=Debug ..
-    cmake -D CMAKE_BUILD_TYPE=Release ..
-    cmake -D CMAKE_BUILD_TYPE=RelMinSize ..
+- `Debug`: no optimization, debugging code enabled
```

Comment:
is this then e.g.

```
cmake --build . --config Debug
```

? Then I'd suggest writing "The four options for `<config>` are:"

Reply:
good idea!